cachelib/allocator/CacheAllocatorConfig.h

```diff
@@ -268,16 +268,21 @@ class CacheAllocatorConfig {
       std::chrono::seconds regularInterval,
       std::chrono::seconds ccacheInterval,
       uint32_t ccacheStepSizePercent);
-  
+
+  // Enable DSA
+  CacheAllocatorConfig& enableDSA(bool useDsa);
```

Comment:
Similar to my comment related DML dependency. We should probably not make it DSA-specific. Instead, this should be something like enableAsyncMovement with callback that specifies how the data is moved. Again, this can be fixed once we have good performance.